### PR TITLE
S2t1 base html css

### PR DESCRIPTION
**Pull request by fatimaseghir**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    font-size: 4rem;
+    text-align: center;
+    margin: 0;
+}
+
+.content p {
+    padding: 1rem;
+    font-size: 1.5rem;
+    margin: 0;
+
```
Comment (henryppercy) [inline css/styles.css:51]: Remove extra line

File: index.html [hunk 2]
```diff
@@ -2,9 +2,30 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Tony the Hawk's Pro Mazer</title>
```
Comment (PNaing107) [inline index.html:5]: Should be Maze not Mazer right?

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    margin: 2rem auto;
+    background-color: var(--light);
+    color: var(--white);
+    width: 17.5rem;
+    font-size: 3rem;
+    border-radius: 0.5rem;
+
+    display: flex;
+    justify-content: center;
+
```
Comment (henryppercy) [inline css/styles.css:70]: Remove extra line

File: index.html [hunk 4]
```diff
@@ -2,9 +2,30 @@
...
+    <div>
+        <p> Welcome to ‘Tony the Hawk’s pro maze’ game.</p>
+        <p> Tony fell over skateboarding into a bush and now he needs to get back to his nest to fix his deck.</p>
+        <p>Press the start button to start the game.</p>
+        <p> Navigate the route back to the nest using your mouse.</p>
+        <p> But beware enemies are out to get Tony, do not bump into them or let them steer you off track.</p>
+    </div>
+</div>
+<button class="enter">ENTER MAZE</button>
+
```
Comment (henryppercy) [inline index.html:27]: Remove extra lines at the bottom on the document.

**Review by RBingham99 — APPROVED** [review]

**Pull request merged**